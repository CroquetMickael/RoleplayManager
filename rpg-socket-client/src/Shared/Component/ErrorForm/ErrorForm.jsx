import React from "react";

const ErrorForm = ({ children }) => (
  <span className="mb-2 text-red-500">{children}</span>
);

export { ErrorForm };
