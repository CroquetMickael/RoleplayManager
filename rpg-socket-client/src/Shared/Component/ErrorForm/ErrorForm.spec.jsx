import React from "react";
import { render } from "@testing-library/react";
import { ErrorForm } from "./ErrorForm";

it("renders correctly", () => {
  const tree = render(<ErrorForm>error</ErrorForm>).container;
  expect(tree).toMatchSnapshot();
});
