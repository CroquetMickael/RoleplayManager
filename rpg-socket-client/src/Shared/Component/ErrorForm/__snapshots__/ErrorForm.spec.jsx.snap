// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <span
    class="mb-2 text-red-500"
  >
    error
  </span>
</div>
`;
